js/releases/index.js

```diff
@@ -50,7 +50,7 @@ class Releases {
    * @returns {Promise} A promise that resolves when the release has been created.
    * @memberof SentryReleases
    */
-  new(release, options) {
+  async new(release, options) {
```

Comment:
why are we adding `async` to this function? `this.execute` should just return a promise directly right, why do we need to wrap it in another promise?

Reply:
ahh TIL that makes sense